### PR TITLE
Added big-picture explanation of filtering

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
This partly addresses #1069

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
CC @fasaxc

File: docs/source/etcd-data-model.rst [hunk 1]
```diff
@@ -265,6 +265,22 @@ The various properties in this object have the following meanings:
 
 .. _security-profile-data:
 
+Traffic Filtering
+~~~~~~~~~~~~~~~~~
+
+Calico implements traffic filtering controlled by security profiles
+and tiered security policies.  Traffic is connection-like: the
```
Comment (fasaxc) [inline docs/source/etcd-data-model.rst]: How about making this `connection-based` instead of `connection-like`?  I think that'd read better.

**Comment by matthewdupre** [comment]
ok to test

**Comment by matthewdupre** [comment]
LGTM - thanks for the contribution :)

**Pull request merged**